### ISSUE TITLE
Viewing logs

### ISSUE THREAD

**Issue by EZarnosky** [Hellowlol/HTPC-Manager#228]
Got a feature request. enable auto update at a selectable interval on the log - i hate hitting refresh.

Also can you add an option to select other logs, the logs from the install 'components'? it would be nice to have a one stop page for all the logs.

**Comment by Hellowlol** [comment]
Im not sure i understand what you mean by "Also can you add an option to select other logs, the logs from the install 'components'? it would be nice to have a one stop page for all the logs."

Care to explain a little further?

**Comment by EZarnosky** [comment]
Provide a drop down that you can select Headphones logs, CouchPotato logs, SickRage logs, etc. Currently you need to open each app/component to veiw the logs

You should be able to dynamically read the location of the log files from the apps config.ini or conf file.

This way i do not have to open SickRage to see what is going on. I just select the SickRage log, set the refresh interval and I can view what is going on in SickRage from HTPC Manager.

**Comment by Hellowlol** [comment]
Ah, i can add the auto refresh, having all the logs at the same place are a no. If you want access to sickrage stuff you will have to use the sickrage module (the log is already there btw)

**Comment by EZarnosky** [comment]
Ok, SickRage was a bad example - you got me there, there is a tab with the log :p

CouchPotato & Headphones (not sure about other apps, I haven't gottent that far yet) don't have a tab for logs.

**Comment by Hellowlol** [comment]
HP is even worse, it isnt supported in the api :D

**Comment by EZarnosky** [comment]
Yeah, I've had a lot of headaches getting Headphones to work right. You would think it is a simple app...

There is a line entry in the Headphones config.ini 'log_dir = ' you tell it the path and it creates a headphones.log in that folder.

**Comment by Hellowlol** [comment]
I think it would be easy to add that to the api if they wanted to. Anyway cp has notfication today, i can replace that with the log?

**Comment by EZarnosky** [comment]
Argh, CP does have 'history' which is the log. I guess I just like standardization of tabs/label (I know that can be difficult when you are adding support for so many different apps)

Honestly I'd love disable all notifications from the apps, instead have HTP Manager be the middleman and notify me via pushbullet (or some other service) when selected events occur.

**Comment by Hellowlol** [comment]
Yeah, but the history is just the notification from cp. I could add a tab with the actual log.

**Comment by EZarnosky** [comment]
I LOVE logs :D

I might not always understand them... but someone out there does :P

I still like the idea of a single log page with a option to select your desired long... but that is what forking is for... at this time I am just to lazy to do that (and my python programming sucks).

**Comment by Hellowlol** [comment]
I get that, i welcome a pr :P Added logs to the api https://github.com/rembo10/headphones/pull/2253

**Comment by EZarnosky** [comment]
Awesome!

**Comment by Hellowlol** [comment]
My pr was merged. Ill add a log to hp.

**Comment by Hellowlol** [comment]
I sent a pr to Mylar regarding logs, hopefully they will merge it

**Comment by Hellowlol** [comment]
It was added to mylar aswell, ill add logs to mylar aswell
